packages/query-core/src/hydration.ts

```diff
@@ -88,6 +89,10 @@ export function defaultShouldDehydrateQuery(query: Query) {
   return query.state.status === 'success'
 }
 
+export function defaultDeserialize(data: unknown) {
+  return data
+}
```

Comment:
didn't find any existing `identity` functions when i did a quick search. lmk if you have one somewhere already

Reply:
don't think we have one